<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contact_widget</class>
 <widget class="QWidget" name="contact_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="load_contact_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_contact_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contact_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark Contact</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="contact_table">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>311</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Contact Start</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Contact End</string>
    </property>
   </column>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>650</width>
     <height>150</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pole_select">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Focus Location</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bounding_box_size">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>130</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Focus Bounding Box Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Location: </string>
   </property>
  </widget>
  <widget class="QLabel" name="current_location_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(250, 250)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="contact_number">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_contact_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flip_contact_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip Contact State</string>
   </property>
  </widget>
  <widget class="QPushButton" name="output_dir_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Dir</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_dir_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
